### ISSUE TITLE
Difficulty correction/adjustment

### ISSUE THREAD

**Issue by TheMusco** [ronpaulcoin/ronpaulcoin#3]
Problem is #'d below. Then proposed solution at end. -mike

Difficulty is currently set to adjust every X blocks, with X being a relatively large number. In the most recent re-target, this scenario played out:
1. Coin generated good market buzz, profitability being high, drew many miners. This caused the network hash rate to quickly go from ~ 400 M/hs to ~1500 M/hs.
2. When the coin hit ~1500 the difficulty then jumped from about 10 to 40. Almost immediately after the network hash rate halved, then halved again (now around 250),
3. The result is an exponentially slower difficulty retarget, and literally no incentive to mine. Even if you're long on the coin, you'd be better off mining a different faster coin, then trading on coinedup for purchasing RPC. 
4. The next time this happens it could be even more dramatic and is not entirely impossible that the coin could be killed.

Proposed solution:
A hard-difficulty retarget of  6-12 hours if possible would be a great solution, if possible. Originally 24 hours seemed to be good but even then a scenario of pump for 1 day then dump would be possible. 

Alternatively, another fairly easy fix would be just lowering the # of blocks for diff adjustment to ~300.

Either of these would have prevented the 1.5G/Hs to 250m/hs (and falling) drop that we had and insulate from a future (and potentially worse) occurrence.

**Comment by ronpaulcoin** [comment]
I agree 100% I think we should do an even smaller amount. I'm thinking 30
minute adjustments

**Comment by ronpaulcoin** [comment]
Sorry, I meant 30 block adjustments, which is 60 minutes.

**Comment by TheMusco** [comment]
30 blocks sounds good
